nautobot_firewall_models/models/nat_policy.py

```diff
@@ -252,10 +252,16 @@ class NATPolicy(PrimaryModel):
         to="nautobot_firewall_models.NATPolicyRule", blank=True, related_name="nat_policies"
     )
     assigned_devices = models.ManyToManyField(
-        to="dcim.Device", through="NATPolicyDeviceM2M", related_name="nat_policies"
+        to="dcim.Device",
+        through="NATPolicyDeviceM2M",
+        related_name="nat_policies",
+        blank=True,
```

Comment:
If you're using automatically created form fields then the default behavior is to raise a ValidationError if the form field for this is blank. Since this project is explicitly defining the form fields for these model fields with `required=False`, this change is effectively a no-op. Based on how the forms were created, adding `blank=True` is the correct and desired behavior for this model field.